trunk/SUAVE/Methods/Aerodynamics/AVL/purge_directory.py

```diff
@@ -0,0 +1,41 @@
+# Tim Momose, December 2014
+
+import os
+from purge_files import purge_files
+
+def purge_directory(path,purge_subdirectories=False):
+	"""
+	Deletes the contents of a directory, then the directory itself.
+	If purge_subdirectories is True, subdirectories of the directory will also
+	be purged, recursively. In this case, the directory specified in the 'path'
+	argument will always be removed at the end of this function. 
+	However, if purge_subdirectories is False, files in 'path' will be deleted,
+	but subdirectories and their contents will be left untouched. In this case, 
+	the directory ('path' argument) will only be removed if it contains no
+	subdirectories.
+	"""
```

Comment:
what's the use case for not removing a whole working file directory?  working/scratch files should all go in one directory so you can purge them with a single shutil.rmtree() command.